AndelaOSP/art-dashboard
# [#160141349]: Stop navigation bar from reappearing

## What does this PR do?
Stops the navigation bar from reappearing on different pages even after it has been closed.

## Description of Task to be completed?
- Add an action and a reducer to save the toggling boolean value in the store
- Refactor the `NavBarComponent` to make use of the boolean value in the store
- Refactor relevant tests and add new ones

## How should this be manually tested?
Head onto the `Assets` page, ensure you close the navbar, click on a single asset to view its details, the navbar should remain closed on the `assetDetails` page as well unless you reopen it.

## What are the relevant pivotal tracker stories?
[#160141349](https://www.pivotaltracker.com/n/projects/2146417/stories/160141349)

## Any background context you want to add?
Prior to this, one would head onto the `Assets` page, close the navbar, but on clicking a single asset to view its details, the navbar would then again reappear on the `assetDetails` page without necessarily being reopened by the user.

## Important notes
N/A

## Packages installed
N/A

## Deployment note
N/A

## Related PRs branch | PR (branch_name) | (pr_link)
Branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()
## Todos
## Screenshots
![screen recording 2018-10-23 at 11 51 am](https://user-images.githubusercontent.com/31322228/47348448-30747600-d6ba-11e8-95f2-a153669aef3f.gif)
